stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/builder/components/Field.java

```diff
@@ -68,6 +68,10 @@ public String getDefaultValue() {
         public Field build() {
             String fieldType = FIELD_TYPE_MAP.get(fieldDescriptor.getType()) != null ? FIELD_TYPE_MAP.get
                     (fieldDescriptor.getType()) : fieldDescriptor.getTypeName();
+            if (fieldType.startsWith(".")) {
+                String[] fieldTypeArray = fieldType.split(("\\."));
```

Comment:
Have we not used these constants in other places? Might be better to introduce constants.